controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorGroupImpl.java

```diff
@@ -82,7 +82,13 @@
 
     void initialize() throws CheckpointStoreException {
 
-        checkpointStore.addReaderGroup(actorSystem.getProcess(), eventProcessorConfig.getConfig().getReaderGroupName());
+        try {
+            checkpointStore.addReaderGroup(actorSystem.getProcess(), eventProcessorConfig.getConfig().getReaderGroupName());
+        } catch (CheckpointStoreException e) {
+            if (!e.getType().equals(CheckpointStoreException.Type.NodeExists)) {
```

Comment:
Let's at least add a warn log message here in the case we don't throw (node exists).